src/posix/platform/netif.cpp

```diff
@@ -1021,6 +1021,70 @@ static void processReceive(otMessage *aMessage, void *aContext)
     }
 }
 
+#if OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE || OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
```

Comment:
I may be just not seeing it, but where do we `#endif` this?

Reply:
after `getIpVersion`

Reply:
Thanks 😄